api_volontaria/apps/position/admin.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+
+from api_volontaria.apps.position.models import (
+    Position,
+    Application,
+)
+
+admin.site.register(Position)
+admin.site.register(Application)
+
+# TODO: determine whether additional classes needed (see volunteer/admin.py)
```

Comment:
Remove this comment since it's no more needed

Reply:
done